sdks/python/apache_beam/io/jdbc.py

```diff
@@ -96,25 +98,37 @@
 
 
 def default_io_expansion_service():
-  return BeamJarExpansionService('sdks:java:io:expansion-service:shadowJar')
+    return BeamJarExpansionService(
+        ':sdks:java:extensions:schemaio-expansion-service:shadowJar')
 
 
-WriteToJdbcSchema = typing.NamedTuple(
-    'WriteToJdbcSchema',
+ReadFromWriteToJdbcSchema = typing.NamedTuple(
+    'ReadFromWriteToJdbcSchema',
+    [
+        ('location', unicode),
+        ('config', bytes)
+    ],
+)
+
+Config = typing.NamedTuple(
+    'Config',
     [
         ('driver_class_name', unicode),
         ('jdbc_url', unicode),
         ('username', unicode),
         ('password', unicode),
         ('connection_properties', typing.Optional[unicode]),
         ('connection_init_sqls', typing.Optional[typing.List[unicode]]),
-        ('statement', unicode),
+        ('write_statement', typing.Optional[unicode]),
+        ('read_query', typing.Optional[unicode]),
+        ('fetch_size', typing.Optional[int]),
+        ('output_parallelization', typing.Optional[bool]),
     ],
 )
 
 
 class WriteToJdbc(ExternalTransform):
-  """A PTransform which writes Rows to the specified database via JDBC.
+    """A PTransform which writes Rows to the specified database via JDBC.
```

Comment:
nit: looks like there's an extraneous whitespace change here.

Please update this docstring (and the Read counterpart) based on our change to table_name/statement/query. It should point out that table_name is required and statement/query are optional, but can be overriden

Reply:
Done.